backend/game.js

```diff
@@ -516,7 +581,7 @@ module.exports = class Game extends Room {
 
       if (addBots) {
         while (this.players.length < this.seats) {
-          this.players.push(new Bot());
+          this.players.push(new Bot(this.isDecadent));
```

Comment:
I though the Bot could not be decadent?

Reply:
Good eye. Fixed https://github.com/dr4fters/dr4ft/pull/990/commits/7d2086c0c339812c5306a64d509af96bd853943b